container/lxd/server_test.go

```diff
@@ -78,6 +78,17 @@ func (s *serverSuite) TestHasProfile(c *gc.C) {
 	c.Check(has, jc.IsFalse)
 }
 
+func (s *serverSuite) TestStorageSupported(c *gc.C) {
+	ctrl := gomock.NewController(c)
+	defer ctrl.Finish()
+	cSvr := s.NewMockServerWithExtensions(ctrl, "storage")
+
+	jujuSvr, err := lxd.NewServer(cSvr)
+	c.Assert(err, jc.ErrorIsNil)
+
+	c.Check(jujuSvr.StorageSupported(), jc.IsTrue)
```

Comment:
If we are testing the positive assertion, don't we need to also test that we notice when it is *not* available? Otherwise we could just have written:
func StorageSupported() { return true }

Or even more sinister:
func StorageSupported { return s.networkSupported }

Reply:
Fair point. Will test the negative.